bundles/best_practices.rst

```diff
@@ -166,6 +166,69 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
```

Comment:
IIRC `sudo` is no longer required

Reply:
I did some searching just now. I cannot find any references to this.. Can you provide a link?

Reply:
Hm, [Same page](https://docs.travis-ci.com/user/reference/trusty/#Container-based-with-sudo%3A-false) says:

> ## Container-based with sudo: false #

>When specifying sudo: false, Travis CI runs each build in a container on a shared host via Docker. The container contents are a pristine copy of the Docker image, as guaranteed by Docker itself.

>**The advantage of running with `sudo: false` is dramatically reduced time between commit push to GitHub and the start of a job on Travis CI**, which works especially well when one’s average job duration is under 3 minutes.

>NOTE: The allowed list of packages that may be installed varies between Precise and Trusty, with the Precise list being considerably larger at the time of this writing. If the packages you need are not yet available on Trusty, it is recommended that you either target dist: precise or sudo: required depending on your needs.

Reply:
It depends on the build distribution, see [here](https://blog.travis-ci.com/2017-08-31-trusty-as-default-status). Newer builds, or builds explicitly using `dist: trusty` will ignore the `sudo:` setting.